### PR TITLE
rename clusterrole and clusterrolebinding

### PR DESCRIPTION
**Pull request by zhiweiyin318**
related issue: https://github.com/open-cluster-management/backlog/issues/3025

### CODE REVIEW

File: manifests/cluster-manager/cluster-manager-registration-clusterrole.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: system:open-cluster-management:{{ .ClusterManagerName }}-registration-controller
```
Comment (qiujian16) [inline manifests/cluster-manager/cluster-manager-registration-clusterrole.yaml:4]: i think  the format is something like open-cluster-management:registration:{{ .ClusterManagerName }}-controller ?

Comment (zhiweiyin318) [reply manifests/cluster-manager/cluster-manager-registration-clusterrole.yaml:4]: done

**Pull request merged**